internal/driver/webui.go

```diff
@@ -117,7 +117,7 @@ func serveWebInterface(hostport string, p *profile.Profile, o *plugin.Options, w
 		},
 	}
 
-	if wantBrowser {
+	if o.UI.IsTerminal() {
```

Comment:
We may need to extend the UI interface with a separate function like CanOpenBrowser() or even with OpenBrowser(url) - I am thinking about environments like a terminal with remote SSH session where it is terminal, but opening a browser GUI against a port of a process user spawns in that terminal is not usually feasible.

Reply:
This doesn't need to be in this PR, I'm just raising a discussion point.

Reply:
Ack, I'll hold off on that for now.